deps.gradle

```diff
@@ -1,10 +1,15 @@
 ext {
     versions = [
-            okHttp             : '3.3.1',
-            support            : '25.1.0',
-            retrofit           : '2.0.2',
+            okHttp             : '3.6.0',
+            support            : '25.2.0',
+            retrofit           : '2.2.0',
             espressoVersion    : '2.2.2',
-            BUILD_TOOLS_VERSION: '25.0.1',
+            BUILD_TOOLS_VERSION: '25.0.2',
+            rxJava: '2.0.1',
+            rxAndroid: '2.0.1',
+            butterknife: '8.5.1',
```

Comment:
butterknife 7.0 is good. I have no trouble with it since its release. Does it make sense to update it?